scripts/download-backup.sh

```diff
@@ -10,8 +10,8 @@ TMP="${TMP:-/ext/tmp/}"
 backupLocation='s3://elife-app-backups/journal-cms/'
 if [ "$(uname -s)" == "Darwin" ]; then
     # Mac OS X
-    selectedMonth="$(date "-v -${delay}d" +%Y%m)"
-    searchPrefix="${selectedMonth}/$(date "-v -${delay}d" +%Y%m%d)"
+    selectedMonth="$(date "-v-${delay}d" +%Y%m)"
```

Comment:
was the previous version broken for you? I followed https://www.freebsd.org/cgi/man.cgi?date where it has a space